extensions/menubar-calendar/src/types/preferences.ts

```diff
@@ -6,6 +6,7 @@ interface Preferences {
   menubarIconStyle: string;
   menubarItemIconStyle: string;
   showEventsInMenubar: string;
+  titleTruncateLength: number;
   largeCalendar: boolean;
   highlightCalendar: boolean;
   showWeekNumber: boolean;
```

Comment:
There is no need for `interface Preferences` anymore since automatically generated in `raycast-env.d.ts`